apps/web/src/components/poll/mobile-poll/time-slot-option.tsx

```diff
@@ -1,6 +1,6 @@
 import { ClockIcon } from "lucide-react";
 import * as React from "react";
-
+import { createBreakpoint } from "react-use";
```

Comment:
I think this is a good solution as well and would be in favour of it, but as I see no CSS modules being leveraged like this elsewhere I tried to avoid it. Can redo it this way however.

Reply:
I need to read up on Tailwind it seems x). Pushed a similar change now.